tiatoolbox/cli/slide_info.py

```diff
@@ -20,54 +20,57 @@
 
 """Command line interface for slide_info."""
 import pathlib
-import sys
-
-import click
 
 from tiatoolbox import utils, wsicore
+from tiatoolbox.cli.common import (
+    TIAToolboxCLI,
+    cli_file_type,
+    cli_img_input,
+    cli_mode,
+    cli_output_path,
+    cli_verbose,
+    no_input_message,
+    prepare_file_dir_cli,
+)
 
-
-@click.group()
-def main():  # pragma: no cover
-    """Define slide_info click group."""
-    return 0
+slide_info_cli = TIAToolboxCLI()
 
 
-@main.command()
-@click.option("--img-input", help="input path to WSI file or directory path")
-@click.option(
-    "--output-path",
-    help="Path to output directory to save the output, default=img_input/../meta",
-)
-@click.option(
-    "--file-types",
-    help="file types to capture from directory, default='*.ndpi', '*.svs', '*.mrxs'",
-    default="*.ndpi, *.svs, *.mrxs, *.jp2",
-)
-@click.option(
-    "--mode",
-    default="show",
-    help="'show' to display meta information only or 'save' to save "
-    "the meta information, default=show",
-)
-@click.option(
-    "--verbose",
-    type=bool,
-    default=True,
-    help="Print output, default=True",
-)
+@slide_info_cli.command()
+@cli_img_input
+@cli_output_path
```

Comment:
@John-P  Is there a way to edit the "help" part or default values for this decorator?

Reply:
Can I just check exactly which decorator you mean? There are many here

Reply:
If so, then simply make the function in `common` into a decorator itself which has a default help argument which you could then override.

Reply:
Please can you provide an example to explain?

Reply:
Actually, got it just trying this. I will let you know if I need help on this.

Reply:
If you want to wrap the decorator, you can do something like this:
```python
def cli_output_path(help:str="Path to output directory to save the output.", *args, **kwargs):
    inner_func = click.option(
        "--output-path",
        help=help,
    )
    return inner_func 
```

Might not be precisely right but that is the gist of it.

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/307/commits/585f88d086b8ebc6a1f65b22a7a59c30dd3ec35d